src/Cache/InMemoryDriver.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tnapf\Config\Cache;
+
+use DateInterval;
+use Psr\SimpleCache\CacheInterface;
+use Tnapf\Config\Exceptions\InvalidCacheKeyException;
+
+class InMemoryDriver implements CacheInterface
+{
+    private array $storage = [];
+
+    public function get(string $key, mixed $default = null): mixed
+    {
+        return $this->has($key) ? $this->storage[$key] : $default;
+    }
+
+    public function set(string $key, mixed $value, DateInterval|int|null $ttl = null): bool
+    {
+        $this->validateKey($key);
+        $this->storage[$key] = $value;
+
+        return true;
+    }
+
+    public function delete(string $key): bool
+    {
+        $this->validateKey($key);
+        unset($this->storage[$key]);
+
+        return true;
+    }
+
+    public function clear(): bool
+    {
+        $this->storage = [];
+
+        return true;
+    }
+
+    public function getMultiple(iterable $keys, mixed $default = null): iterable
```

Comment:
Does a doc block with `@param string[] $keys` mess with the `iterable` typing? If not, please add it